src/angular/tooltip/tooltip-wrapper.spec.ts

```diff
@@ -173,6 +173,24 @@ describe('SbbTooltipWrapper', () => {
       // throw if we have any timers by the end of the test.
       fixture.destroy();
     }));
+
+    it('should emit event on dismissing tooltip', fakeAsync(() => {
+      let event: SbbTooltipChangeEvent | null = null;
+      component.tooltip.dismissed.subscribe((e) => (event = e));
+
+      component.tooltip.show();
+      tick();
+      fixture.detectChanges();
+      component.tooltip.hide();
+      tick();
+      fixture.detectChanges();
```

Comment:
This should fix the unit test.
```suggestion
      fixture.detectChanges();
      tick();
```